src/ui/marker.js

```diff
@@ -604,4 +607,29 @@ export default class Marker extends Evented {
     getPitchAlignment() {
         return this._pitchAlignment;
     }
+
+    /**
+     * Sets the current `color` property of the marker.
+     * @param {string} [color] Sets the `color` property of the marker. If alignment is 'auto', it will automatically match `rotationAlignment`.
+     * @returns {Marker} `this`
+     */
+    setColor(color: ?string) {
+        if (color) {
+            this._color = color;
+        } else {
+            this._color = this._defaultColor;
+        }
+
+        this._background.setAttributeNS(null, 'fill', this._color);
```

Comment:
will this work for Markers with a user defined element? In that case the default marker is never created so this._background wouldn't exist right?

Reply:
I added a line `if ('undefined' !== typeof this._background) {`.
Actually, I am not sure that if user use custom marker element, do we need something to describe it in this method like `console.log()` or so.